ui/components/tsconfig.json

```diff
@@ -3,6 +3,9 @@
   "include": ["src"],
   "compilerOptions": {
     "outDir": "./dist",
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "paths": {
+      "@perses-dev/components": ["./src"]
```

Comment:
Allows embedded stories to reference source code at the top level without thinking too hard about it. See the storybook README for a big explanation of why we're doing this.

Reply:
This makes sense, basically bypass the build and go directly to the files as if it were part of the project.